web2.0/src/Components/weather-card/WeatherCard.jsx

```diff
@@ -0,0 +1,10 @@
+const WeatherCard = ({ weatherData, weatherDataLabel }) => {
```

Comment:
I recommend reducing your weather-card components, but extending what you do keep to serve multiple purposes.

For example, compare this `<WeatherCard />` and [`<WindSpeedCard />`](web2.0/src/Components/weather-card/WindSpeedCard.jsx), the only differences is the className value and the string 'mph'.

In React, a component should serve a specific purpose like a class in an OOP language. You can think of the hierarchy of parent-child in a similar way when designing/wire-framing the component structure.

I find that such refactors are one of the more challenging aspects of React. It's also a big part of what makes React very powerful. It gets easier with time and practice. I strongly encourage you to reduce your components to at most 2 or 3 types of weather cards.